src/views/OfferTable/OfferTable.vue

```diff
@@ -0,0 +1,221 @@
+<template>
+  <div class="offer-header-container"></div>
+  <div class="offer-table">
+    <div class="table-title">
+      <h1> My Offers</h1>
+    </div>
+    <table style="width: 100%;" v-if="offers.length !==0 ">
+      <thead>
+      <tr>
+        <th>
+          Official Name <i class="bi bi-sort-alpha-down" aria-label='Sort Icon'></i>
+        </th>
+        <th>
+          Field <i class="bi bi-sort-alpha-down" aria-label='Sort Icon'></i>
+        </th>
+        <th>
+          Price <i class="bi bi-sort-alpha-down" aria-label='Sort Icon'></i>
+        </th>
+        <th>
+          Sent Date <i class="bi bi-sort-alpha-down" aria-label='Sort Icon'></i>
+        </th>
+        <th>
+          Status <i class="bi bi-sort-alpha-down" aria-label='Sort Icon'></i>
+        </th>
+      </tr>
+      </thead>
+      <tbody>
+      <tr  v-for="offer in offers" :key='offer'>
+        <td style="width: 30%;text-align: center; color: black">{{offer['officialName']}}</td>
+        <td style="width: 20%;text-align: center; color: #42474d">Field</td>
+        <td style="width: 20%;text-align: center; color: #42474d">{{offer['bidPrice']}}</td>
+        <td style="width: 20%;text-align: center; color: #42474d">{{offer['offerDate'] ? formatDate(offer['offerDate']) : ''}}</td>
+        <td style="width: 10%;text-align: center; color: #42474d">{{offer['offerStatus'] ? offer['offerStatus'].nameCt : ''}}</td>
+      </tr>
+      </tbody>
+    </table>
+    <v-pagination
+        :disabled = "offers.length === 0"
+        v-model="page"
+        :length="pageCount"
+        :showFirstLastPage="true"
+        :size="countPerPage"
+        @update:modelValue="pageChange()"
+    ></v-pagination>
+
+
+    <div class="blank-table" v-if="offers.length === 0">
+      <div class="no-offers">There are no sent offers</div>
+    </div>
+  </div>
+
+</template>
+
+<script>
+import {getAuthenticatedHeaders,logout} from "@/common_functions";
+import {API_BASE_URL} from "@/const_config.js";
+export default {
+    props: ['tender'],
+    data() {
+      return {
+        countPerPage: 5,
+        offersCount: 0,
+        offers: [],
+        page: 1
+      }
+    },
+     computed: {
+        pageCount () {
+          return (this.offersCount%this.countPerPage !== 0) ? this.offersCount/this.countPerPage+1 : this.offersCount/this.countPerPage}
+      },
+    async beforeMount() {
+      if(this.tender){
+        this.offers = await fetch (API_BASE_URL+ '/tenders/'+this.tender.id+'/offers', {
+          method: 'GET',
+          node: 'cors',
```

Comment:
you can move  _node: 'cors', cache: 'no-cache', credentials: 'same-origin',_ to config to avoid duplication